### DISCUSSION TITLE
GitHub copilot works abnormal.

### DISCUSSION THREAD

**Discussion by javaice** [community/community#52234] [Copilot Conversations]
### Select Topic Area

Show & Tell

### Body

I'm using GitHub Copilot for weeks,  and recently I've found the Q&A mode works abnormally.
Please check this Q&A approach, I did the same question and got expectation answer this morning, and now when I notice the copilot works abnormally I asked the questions again and the answer always points to a URL (incorrect or invalid) but for some simple question the answer still functional like:

// q: what's the length of "D68D205B375D717A331D3EA1F29207EF9C9E80A7"?
// a: 40

I want my smart copilot back please help.

![image](https://user-images.githubusercontent.com/1538943/230560995-a45b2b32-ada8-41ab-bf77-85525b5cc748.png)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @javaice ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
